internal/services/machinelearning/machine_learning_compute_instance_resource.go

```diff
@@ -123,9 +120,15 @@ func resourceComputeInstance() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
```

Comment:
Double checking here since per our guides for schema design [we don't expose the `enabled` field for blocks](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/schema-design-considerations.md#features-that-are-toggled-by-the-property-enabled). Can't `ForceNew` be removed from the block and enabling/disabling be toggled by the presence of the block?

Reply:
I have updated the contributor guidelines as discussed internally earlier this week https://github.com/hashicorp/terraform-provider-azurerm/pull/27954